PROVESFlightControllerReference/test/unit-tests/test_PacketProcessor_Validator.cpp

```
@@ -0,0 +1,87 @@
+    Packet p{0u, 10u, 0x2200B000u, {}};  // bypass opcode
+    auto res = validatePacket(p, 10u, 5u);
+    EXPECT_EQ(res, PacketValidator::Status::Bypass);
+}
+
+TEST(PacketValidatorTest, BypassTellJokeOpCode) {
+    Packet p{0u, 10u, 0x10065000u, {}};  // bypass opcode
+    auto res = validatePacket(p, 10u, 5u);
+    EXPECT_EQ(res, PacketValidator::Status::Bypass);
+}
+
+TEST(PacketValidatorTest, BypassSpiInvalid) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)